.github/workflows/swift_package_test.yml

```diff
@@ -58,5 +58,5 @@ jobs:
           done
     - name: Pre-build
       run: ${{ inputs.pre_build_command }}
-    - name: Build
+    - name: Build / Test
```

Comment:
`/` is what GitHub itself uses in names like `test / api-breakage (pull_request)`.
Should we avoid using `/` to not ambiguate where that slash is coming from?